include/yaml-cpp/node/detail/node_data.h

```diff
@@ -114,7 +114,7 @@ class YAML_CPP_API node_data {
   mutable std::size_t m_seqSize;
 
   // map
-  typedef std::map<node*, node*> node_map;
+  typedef std::list<std::pair<node*, node*> > node_map;
```

Comment:
My bad, I didn't make sure the style used C++ 11. Please try again at HEAD, it should adjust it to >>